### DISCUSSION TITLE
Could be interesting a little description of the parameters in the AutoComplete Suggestion Box?

### DISCUSSION THREAD

**Discussion by rafa4mat** [WinEdt-Team/WinEdt-Support#77] [1.3 Ideas and features requests]
When you have the list of suggestions,... when you use a command a few times and can´t remember the order of the parameters... may be interesting a little remember.

Could be interesting a little description on the rigth of the Suggestion Box?, similar to 

`\definecolor{'bullet'}{'bullet'}{'bullet','bullet','bullet'}          \definecolor{name color}{model}{0-1,0-1,0-1}`
or
`\definecolor{'bullet'}{'bullet'}{'bullet','bullet','bullet'}          \definecolor{name color}{model}{0-1,0-1,0-1} % [requires xcolor package]`

**Comment by rafa4mat** [comment]
I answer myself.

It´s already possible (even better) typing a word instead of a bullet, in this way:

\definecolor{name color}{model}{0-1,0-1,0-1}

Excuse me for the Question, you can delete it if you want.

**Reply by Alex-Simonic** [reply]
No need for apologies. If I was 40 years younger I would probably do something about it. As it is, WinEdt makes a few hundred dollars per month and I am not very motivated to do anything about it. For some users going freeware IA empowered (or written) alternatives to WinEdt is what you should do. No hard feelings on my part. Times are changing again... and I am ready to sail into the sunset. Nothing to be afraid of. Or is it?

**Comment by rafa4mat** [comment]
Nothing :-)

Thank you very much, Alex.

Sometimes I think that a donate button could be interesting in WinEdt page for those who like and appreciate your effort and work, and to promote upgrades. But I understand what you tell about "Times are changin again".

Are there any people helping you improving/updating WinEdt?.

Have a good sailing! :-)
